demos/README.md

```diff
@@ -2,12 +2,39 @@
 
 ## Technical requirements
 
-- **FFmpeg**
+- **FFmpeg** (FFmpeg6 on Linux, FFmpeg 7 on MacOS)
 - **Gstreamer**
 - NodeJS + npm
 
 Before running demos, install JS dependencies with:
 
+```console
+npm install
+```
+LiveCompositor should run on any computer with modern GPU, but if you want to check detailed requirements, visit [requirements section in docs](https://compositor.live/docs/deployment/requirements).
+
+### MacOS installation guide
+
+1. Install FFmpeg 7
+```console
+brew install ffmpeg
+```
+
+2. Install GStreamer
+
+- Go to GStreamer website, download and open runtime installers (we tested 1.24 release, but should work on the others as well).
```

Comment:
Will this installer install all gst plugins? Don't you need to specify which plugins need to be installed?